src/components/QueryEditor.tsx

```diff
@@ -45,7 +45,7 @@ export class QueryEditor extends PureComponent<Props, State> {
   };
 
   render() {
-    const projects = this.props.datasource.fetchProjects();
```

Comment:
This method is renamed to clarify that it is simply a property access, no data fetching is involved.